src/prefect/utilities/callables.py

```diff
@@ -312,18 +313,19 @@ def process_v1_params(
 def create_v1_schema(
     name_: str, model_cfg: type[Any], model_fields: Optional[dict[str, Any]] = None
 ) -> dict[str, Any]:
+    import pydantic.v1 as pydantic_v1
```

Comment:
When would the schema be v1 for our client?

Reply:
discussed sync, the key point was that this is bc we support types from the v1 backport in pydantic v2